types/chrome/index.d.ts

```diff
@@ -2627,9 +2627,9 @@ declare namespace chrome.downloads {
         /** Optional. Extra HTTP headers to send with the request if the URL uses the HTTP[s] protocol. Each header is represented as a dictionary containing the keys name and either value or binaryValue, restricted to those allowed by XMLHttpRequest.  */
         headers?: HeaderNameValuePair[] | undefined;
         /** Optional. The HTTP method to use if the URL uses the HTTP[S] protocol.  */
-        method?: string | undefined;
+        method?: "GET" | "POST";
```

Comment:
you should add `undefined` as an option here

```suggestion
        method?: "GET" | "POST" | undefined;
```

Reply:
Just add `undefined` and you're good!
```suggestion
        method?: "GET" | "POST" | undefined;
```